components/options_resolver.rst

```diff
@@ -842,6 +842,29 @@ the option::
 This closure receives as argument the value of the option after validating it
 and before normalizing it when the option is being resolved.
 
+Ignore not defined Options
```

Comment:
Sometime there an uppercase, sometime not.
Can you explain when we have to use it ?

Thanks

Reply:
It's called "title case" or "headline case" and the rules are "complicated" (i.e. we can't define a rule). There are several styles. Here you can see some examples: https://en.wikipedia.org/wiki/Title_case